### ISSUE TITLE
[Tooltip] - Make docs with more examples

### ISSUE THREAD

**Issue by MrVorgoth** [livechat/design-system#816]
### Description

Better docs would be helpful to get developer started with work eg. there is really good documentation about `<Tooltip />` component and I got instructions from designer to use `Interactive tooltip` - at first I thought that it would be the same component as tooltip with some features, but I was wrong. I couldn't find about it inside `storybook`, then I needed to dig into code to see what this component is doing and later it turned out that it's only a "body" to actual `Tooltip` component. It would be really helpful to have such info inside docs.

To sum it up I think it would be great idea that when you develop some new component in DS it's mandatory to adjust docs for it as well, because other developers are using DS and it would be more than helpful for them to have such info right away inside the docs with example of usage :)
